server/models.py

```diff
@@ -97,5 +97,5 @@ def verify_auth_token(token):
             return None  # valid token, but expired
         except BadSignature:
             return None  # invalid token
-        user = User.query.get(data['id'])
-        return user
\ No newline at end of file
+        user = User.objects.get(email=data['id'])
```

Comment:
The only thing that i have any concern about. Just want to make sure that this is the intended behavior of verifying the token.